transport/src/main/java/io/netty/channel/DefaultEventLoop.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.netty.channel;
 
+import io.netty.channel.local.LocalChannel;
+import io.netty.channel.local.LocalServerChannel;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.netty.channel.local.LocalServerChannel'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)